text/0124-documents-api.md

```diff
@@ -49,13 +53,20 @@ Meilisearch orders documents depending on the hash of their id.
 
 Unique identifier of an index.
 
-##### 3.1.1.2. Query Parameters
+##### 3.1.1.2. Parameters
 
-| Field                    | Type                     | Required |
-|--------------------------|--------------------------|----------|
-| `offset`                 | Integer / `null`         | false    |
-| `limit`                  | String / `null`          | false    |
-| `fields`                 | String / `null`          | false    |
+The following parameters need to be send as:
+- Query parameter for the `GET` - `indexes/:index_uid/documents` route.
+- Json body for the `POST `indexes/:index_uid/documents/fetch` route.
+
+| Field                    | Type                      | Required |
+|--------------------------|---------------------------|----------|
+| `offset`                 | Integer                   | false    |
+| `limit`                  | Integer                   | false    |
+| `fields`                 | Array of Strings / `null` | false    |
+| `filter`                 | filter / `null`           | false    |
+
+In the case of the query parameter, as always, the `filter` can only be a string, while it can be either a string, an array of strings, or an array of array of strings for the json body.
```

Comment:
```suggestion
In the case of the query parameter, as always, the `filter` can only be a string, while it can be either a string, an array of strings, or an array of array of strings for the JSON body.
```